app/controllers/project_sboms_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ProjectSbomsController < ApplicationController
+  helper ProjectsHelper
+
+  before_action :set_project_or_fail
+
+  def index
+    @project_sbom = @project.sboms&.first
```

Comment:
`sboms` will be an enumerable, which is never nil. `[].first` is always nil. So we don't need `&.` here.